src/_includes/macros/filters.html

```diff
@@ -0,0 +1,384 @@
+            #}
+            {% set from_date = selected_filters_for_field('range_date_gte')[0] %}
+            {% set from_date_formatted = from_date | date('%m/%d/%Y') if from_date %}
+            <div class="form-l_col form-l_col-1-2">
+                <label for="{{ options.id_range_date_gte }}">
+                    From <span class="date_label">(YYYY-MM)</span>:
+                </label>
+                <input class="js-filter_range-date js-filter_range-date__gte"
+                       id="{{ options.id_range_date_gte }}"
+                       name="filter_range_date_gte"
+                       type="text"
+                       value="{{ from_date_formatted if from_date_formatted else '' }}"
+                       placeholder="YYYY-MM">
+            </div>
+            {#
+                ===============
+                "To" date input
+                ===============
+            #}
+            {% set to_date = selected_filters_for_field('range_date_lte')[0] %}
+            {% set to_date_formatted = to_date | date('%m/%d/%Y') if to_date %}
+            <div class="form-l_col form-l_col-1-2">
+                <label for="{{ options.id_range_date_lte }}">
+                    To <span class="date_label">(YYYY-MM)</span>:
+                </label>
+                <input class="js-filter_range-date js-filter_range-date__lte"
+                       id="{{ options.id_range_date_lte }}"
+                       name="filter_range_date_lte"
+                       type="text"
+                       value="{{ to_date_formatted if to_date_formatted else '' }}"
+                       placeholder="YYYY-MM">
+            </div>
+          </div>
+        </div>
+    {% endif %}
+{% endfor %}
+    </div> <!-- END .form-l__flush -->
+
+    <div class="form-actions form-actions__right-on-med">
+        <input class="btn btn__warning btn__link form-actions_item js-form_clear"
+               type="button"
+               value="Reset filters">
+        <input class="btn form-actions_item"
+               type="submit"
+               value="{{ options.submit_label }}">
+    </div>
+
+</form>
+{% endmacro %}
+
+
+{# A helper macro for filters()
+   ========================================================================== #}
+{% macro _filter_checkbox(field, value, label, checked=false) %}
+    <input class="custom-input"
+           type="checkbox"
+           name="filter_{{ field }}"
+           value="{{ value }}"
+           {%- if is_filter_selected(field, value) or checked %}
+           checked
+           {%- endif -%}
+           >
+        {{ label if label else value }}
+{% endmacro %}
+
+{# A helper macro for filters()
+   ========================================================================== #}
+{% macro _filter_option(field, value) %}
+    <option value="{{ value }}"
+        {%- if is_filter_selected(field, value) %}
+            selected
+        {%- endif -%}
+        {{ '>' | safe ~ value }}
```

Comment:
I think this was a fix for the filters at some point because of how it rendered in the DOM.